### ISSUE TITLE
`expected string or function, found none` when heading is present

### ISSUE THREAD

**Issue by bluebear94** [Marmare314/lemmify#5]
Minimal working example:

```
#import "@preview/lemmify:0.1.1": *

#let (
  theorem, lemma, corollary,
  remark, proposition, example,
  proof, rules: thm-rules
) = default-theorems("thm-group", lang: "en")

#show: thm-rules

= Introduction

#theorem[
  If $w_1, w_2, dots, w_k >= 0$, then $L$ is star-free.
]<starfree>
```

**Comment by Marmare314** [comment]
The problem is that you did not set a numbering for headings. So because the default style uses heading numberings this error occurs. You could either use `#set heading(numbering: "1.1")` or use a different theorem style (for example `thm-numbering-linear`).

I'm not sure if there is any good alternative to this error, since at least to me, it is not clear what else should happen in this situation. But a more explicit error message could definitely be added.

**Comment by Marmare314** [comment]
Fixed by b320958.
